src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -112,37 +116,40 @@ permissionStatus.onchange = () => {
 };
 ```
 
-## 🆕 The new image-focused portion of the Asynchronous Clipboard API {: #images }
+## The new image-focused portion of the Asynchronous Clipboard API {: #images }
```

Comment:
I think this title is wordy, but I don't have another idea.

Reply:
As you said in another PR - it won't be new forever ;)

Reply:
The—at the time of writing—new image-focused portion of the Asynchronous Clipboard API 🤣